config/default_records/request_types/018_limber_lcm_triomics_request_types.wip.yml

```diff
@@ -0,0 +1,12 @@
+# Request types for LCM Triomics DNA WGS and EMSeq
+---
+limber_lcm_triomics:
```

Comment:
Feel free to disagree or check with Abdullah / customers, but I think an appropriate name for this would be lcm_triomics_emseq

Because I think LCM Triomics refers to the whole suite of pipelines (See [doc](https://docs.google.com/document/d/1HcQFBiFNBCYqKC6FDYGJX6FV-XbLYxrRJfJV_L14hj4/edit?usp=sharing) for 5-bullet point list)

and this request type is specifically for the first bit up to ADP Lig, and also the EM-Seq branch that comes after that.